LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 sleddog
```

Comment:
Not sure what the rule is... If full name is required changed this to "Devin Gray" as opposed to "sleddog"